internal/command/postgres/create.go

```diff
@@ -403,6 +406,7 @@ type ClusterParams struct {
 	Manager    string
 	ForkFrom   string
 	Autostart  bool
+	ImageRef   string
```

Comment:
Just curious—if I understand right, this shadows the `ImageRef` field in the embedded `PostgresConfiguration` (that gets set from the `--image-ref` option). Is this intended?

Reply:
Yeah, this is wrong.  Let me fix.